### ISSUE TITLE
Result for RansacLine is wrong

### ISSUE THREAD

**Issue by dlorenz70** [accord-net/framework#1587]
**What would you like to submit?** (put an 'x' inside the bracket that applies)

  - [ ] question
  - [x] bug report
  - [ ] feature request

**Issue description**
I try to use RansacLine for a line which should have a non-zero intercept:

            Accord.Point[] points = new Accord.Point[500];
            for (int i = 0; i < points.Length; i++)
            {
                float x = i;
                float y = 100;
                points[i] = new Accord.Point(x, y);
            }

            RansacLine target = new RansacLine(0.8, 0.99);

            Line actual = target.Estimate(points);

You would expect that the actual line has Intercept=500 and Slope=0 but the values are completely different (Slope = 0.9554685, Intercept = 0.2950929)

**Comment by justingruenberg** [comment]
I've ran into the same issue.  The inliars are still correct, and you can use those with the OrdinaryLeastSquares class to get your answer.

**Comment by dlorenz70** [comment]
Thanks a lot. This works for me.
So it is a documentation issue?

**Comment by justingruenberg** [comment]
No, it's an issue with using SVD to do the line fitting.  This would probably fixed by normalizing the points and the adjusting the output at the end.  Or replacing SVD with plain and simple least squares.

**Comment by AirLancer** [comment]
I can confirm the issue. It is really simple to reproduce with 3 points only:
```
var points = new List<Accord.Point>();
points.Add(new Accord.Point(0, 0));
points.Add(new Accord.Point(0.5f, 0.5f));
points.Add(new Accord.Point(1,1));
var ransacLine = new RansacLine(0.1, 0.9);
var line = ransacLine.Estimate(points);
```
The slope obviously ought to 1 and the intercept at 0, but the results are:
slope: 0.6644391, interecpt: -0.7473424.
When using only 2 points the results are correct.
Also, changing the point [1,1] to [1000,1000] improves the result: 
slope: 0.999999762, intercept: 0.000707459869

I'm targeting .net framework 4.7.2 and am using Accord in version 3.8.0.
